concepts/psp-replacement/restricted-pod-spec.yaml

```diff
@@ -0,0 +1,85 @@
+# This is a sample Pod specification which contains only entries relevant to meet the restricted Pod Security Standard from Kubernetes
+# Details about Pod Security Standards can be found here: https://kubernetes.io/docs/concepts/security/pod-security-standards
+# Information about Pod Security Admission, an Admission Controller to enforce Pod Security Standards, can be found here: https://kubernetes.io/docs/concepts/security/pod-security-admission/
+apiVersion: v1
+kind: Pod
+metadata:
+  name: <restricted-compliant-spec>
+  annotations:
+    container.apparmor.security.beta.kubernetes.io: runtime/default # Mandatory
+spec:
+  securityContext:
+    runAsUser: 10001 # Mandatory. Any number higher than 10000
+    runAsGroup: 30001 # Mandatory. Any number higher than 10000
+    runAsNonRoot: true # Mandatory
+    seccompProfile: # Mandatory. This option can also be set on container level but I recommend to set it on Pod level and leave it undefined on container level.
+      type: RuntimeDefault
+    sysctls: # Only the following calls are allowed. If it isn't necessary to modify kernel attributes (which usually should be the case), the whole sysctl-section has to be removed.
+      - name: kernel.shm_rmid_forced,
+        value: ''
+      - name: net.ipv4.ip_local_port_range,
+        value: ''
+      - name: net.ipv4.ip_unprivileged_port_start,
+        value: ''
+      - name: net.ipv4.tcp_syncookies,
+        value: ''
+      - name: net.ipv4.ping_group_range,
+        value: ''
+  hostNetwork: false # Optional. The default is false if the entry is not there.
+  hostPID: false # Optional. The default is false if the entry is not there.
+  hostIPC: false # Optional. The default is false if the entry is not there.
+  volumes: # Only the volume types in the following list are allowed. If no volumes needed, the whole volumes-section has to be removed. Otherwise replace the brackets with the volumes needed.
```

Comment:
```suggestion
  volumes: # Only the volume types in the following list are allowed. If no volumes needed, the whole volumes-section has to be removed. Otherwise, replace the brackets with the volumes needed.
```